cmd/mta.go

```diff
@@ -193,6 +213,22 @@ var getBuildParametersCmd = &cobra.Command{
 	SilenceErrors: true,
 }
 
+// getGlobalParametersCmd get global parameters from mta
+var getGlobalParametersCmd = &cobra.Command{
+	Use:   "globalParameters",
```

Comment:
Since in the mta.yaml it's just called `parameters` and not `global-parameters` please change the command name to `parameters` instead of `globalParameters` (also the documentation, messages etc)